lib/mollie/customer/subscription.rb

```diff
@@ -64,6 +66,14 @@ def times=(times)
         @times = times.to_i
       end
 
```

Comment:
Can you add the `timesRemaining` parameter as well?

Reply:
👍